Analyzer/src/TwoLepAnalyzer.cc

```diff
@@ -0,0 +1,442 @@
+                    }
+                }
+                for(int b=0; b < NGoodBJets_pt30; b++)
+                {
+                    if(b != used_jet)
+                    {
+                        double mbl2 = (lep2 + GoodBJetsVec_pt30.at(b)).M();
+                        if( abs(mbl2 - 105) < mbl2diff)
+                        {
+                            Mbl2 = mbl2;
+                            mbl2diff = abs(mbl2 - 105);
+                        }
+                    }
+                }
+            }
+            else if (NGoodBJets_pt30 == 1)
+            {
+                if ( abs((lep1 + GoodBJetsVec_pt30.at(0)).M() - 110) <= abs((lep2 + GoodBJetsVec_pt30.at(0)).M() - 110) )
+                {
+                    Mbl1 = (lep1 + GoodBJetsVec_pt30.at(0)).M();
+                    for (int j = 0; j < Jets.size(); j++)
+                    {
+                        double mbl2 = (lep2 + Jets.at(j)).M();
+                        if (abs(mbl2 - 105) < mbl2diff && !GoodBJets_pt30.at(j))
+                        {
+                            Mbl2 = mbl2;
+                            mbl2diff = abs(mbl2 - 105);
+                        }
+                    }
+                }
+                else
+                { 
+                    Mbl2 = (lep2 + GoodBJetsVec_pt30.at(0)).M();
+                    for (int j = 0; j < Jets.size(); j++)
+                    {
+                        double mbl1 = (lep1 + Jets.at(j)).M();
+                        if (abs(mbl1 - 105) < mbl1diff && !GoodBJets_pt30.at(j))
+                        {
+                            Mbl1 = mbl1;
+                            mbl1diff = abs(mbl1 - 105);
+                        }
+                    }
+                }
+            }
+            else            
+            {
+                for(int j=0; j < Jets.size(); j++)
+                {
+                    double  mbl1 = (lep1 + Jets.at(j)).M();
+                    if( abs(mbl1 - 105) < mbl1diff)
+                    {
+                        Mbl1 = mbl1;
+                        mbl1diff = abs(mbl1 - 105);
+                        used_jet = j;
+                    }
+                }
+                for(int j=0; j < Jets.size(); j++)
+                {
+                    if( j != used_jet)
+                    {                    
+                        double mbl2 = (lep2 + Jets.at(j)).M();
+                        if( abs(mbl2 - 105) < mbl2diff)
+                        {
+                            Mbl2 = mbl2;
+                            mbl2diff = abs(mbl2 - 105);
+                        }
+                    
+                    }
+                }
+            }
+                
+        }
+        // cutmap booleans
+        bool pass_general =  passMadHT && passBlind;
+        bool pass_jgeneral = passMadHT && passBlind && JetID;
+        bool pass_1l = NGoodLeptons==1;
+        bool is_elec = false;
+        bool is_muon = false;
+        if(pass_1l) {
```

Comment:
A ternary if statement is perfect for this situation:
bool is_elec = (pass_1l) ? GoodLeptons[0].first=="e" : false